sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/CalculateSchemas.java

```diff
@@ -0,0 +1,93 @@
+package org.apache.beam.sdk.io.gcp.bigquery;
+
+/** Created by relax on 4/30/17. */
```

Comment:
Remove this thing (and maybe disable the IDE setting to generate it); and place license at top of the file